switchtec_dma.c

```diff
@@ -598,6 +670,20 @@ static int disable_channel(struct switchtec_dma_chan *swdma_chan)
 	return 0;
 }
 
+static int switchtec_dma_pause(struct dma_chan *chan)
+{
+	struct switchtec_dma_chan *swdma_chan = to_switchtec_dma_chan(chan);
+
+	return pause_channel(swdma_chan);
+}
+
+static int switchtec_dma_resume(struct dma_chan *chan)
+{
+	struct switchtec_dma_chan *swdma_chan = to_switchtec_dma_chan(chan);
+
+	return resume_channel(swdma_chan);
```

Comment:
Why bother with the thin wrapper? Why not just open code pause_channel and resume_channel?

Reply:
There are some other similar functions like reset_channel which take struct switchtec_dma_chan as parameter, I'd like to keep these functions in a similar style, so I declare it like this and add a wrapper to them for callbacks.

Reply:
That's just a waste of lines of code and makes it harder to follow. Uniformity is just confusing if there's no actual good reason for a bunch of code.

Reply:
Updated.